staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_common.go

```diff
@@ -151,6 +154,8 @@ func (c *controllerCommon) AttachDisk(isManagedDisk bool, diskName, diskURI stri
 	}
 
 	klog.V(2).Infof("Trying to attach volume %q lun %d to node %q.", diskURI, lun, nodeName)
+	c.diskAttachDetachMap.Store(strings.ToLower(diskURI), "")
```

Comment:
Can we also store the operation which we are performing in the map so that we can distinguish it for later ?

Reply:
fixed.